src/StackAdmin/Azs.Subscriptions/Module/Azs.Subscriptions/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProviderOffer.ps1

```diff
@@ -72,21 +72,21 @@ function Get-AzsDelegatedProviderOffer
     $ErrorActionPreference = 'Stop'
 
     $NewServiceClient_params = @{
-        FullClientTypeName = 'Microsoft.AzureStack.Management.Subscriptions.SubscriptionsManagementClient'
```

Comment:
Why did you remove Management from the name of the client?

Reply:
Alignment with Azure :)

Reply:
What do you mean in alignment with Azure? 

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Generated/ComputeManagementClient.cs